control/installation.xml

```diff
@@ -1,25 +1,37 @@
 <?xml version="1.0"?>
 <productDefines xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
 
-<!--
-Work around for the text domain
-textdomain="control"
--->
-
-    <textdomain>control</textdomain>
-
-    <texts>
-      <xen_host_role>
-        <!-- TRANSLATORS: a label for a system role -->
-        <label>Xen Virtualization Host</label>
-      </xen_host_role>
-      <xen_host_role_description>
-        <label>• Bare metal hypervisor and tools
-• No separate /home partition</label>
-      </xen_host_role_description>
-    </texts>
-
-    <update>
+  <!--
+      Work around for the text domain
+      textdomain="control"
+  -->
+
+<textdomain>control</textdomain>
+
+<texts>
+  <xen_host_role>
+    <!-- TRANSLATORS: a label for a system role -->
+    <label>XEN Virtualization Host</label>
+  </xen_host_role>
+  <xen_host_role_description>
+    <label>• Kernel-based hypervisor and tools
+• Dedicated /var/lib/libvirt partition
+• Disable firewall, kdump services</label>
+  </xen_host_role_description>
+</texts>
+
+  <globals>
+    <enable_autologin config:type="boolean">false</enable_autologin>
+    <enable_firewall config:type="boolean">false</enable_firewall>
+    <firewall_enable_ssh config:type="boolean">false</firewall_enable_ssh>
+    <displaymanager_shutdown>root</displaymanager_shutdown>
+    <enable_sshd config:type="boolean">true</enable_sshd>
+    <dhclient_set_hostname config:type="boolean">true</dhclient_set_hostname>
+    <enable_kdump config:type="boolean">false</enable_kdump>
+    <default_ntp_setup config:type="boolean">true</default_ntp_setup>
+  </globals>
```

Comment:
I worry that this won't work as you want. It modify globals always and not just when you select role. If you want to apply it only when role is selected, it have to be in system role like
```
<system_role>
  <globals>
     ...
  </globals>
```